osbenchmark/builder/installers/docker_installer.py

```diff
@@ -6,11 +6,11 @@
 from jinja2 import select_autoescape
 
 from osbenchmark.builder.installers.installer import Installer
-from osbenchmark.exceptions import InvalidSyntax, SystemSetupError
 
 
 class DockerInstaller(Installer):
-    def __init__(self, pci, node_name, ip, http_port, node_root_dir, distribution_version, benchmark_root):
+    def __init__(self, pci, executor, node_name, ip, http_port, node_root_dir, distribution_version, benchmark_root, ):
```

Comment:
nit: tailing comma seems preferred for multiple line argument lists ![here](https://peps.python.org/pep-0008/#when-to-use-trailing-commas) ...

Reply:
Is this from an intermediary commit? The init signature currently looks like this:
```
def __init__(self, provision_config_instance, executor):
```

Reply:
Ah, ok, ignore it.